src/unity.c

```diff
@@ -573,10 +573,7 @@ void UnityAssertEqualIntArray(UNITY_INTERNAL_PTR expected,
 
     RETURN_IF_FAIL_OR_IGNORE;
 
-    if (num_elements == 0)
```

Comment:
Our "coding standard" (in quotes, because clearly we don't do a good job of telling people what it is) is to always have braces and to always have the THEN portion of an if statement on a newline.